### PR TITLE
Add 2 Lookups required for Ipa Role

### PR DESCRIPTION
**Pull request by rsuplina**

### CODE REVIEW

File: plugins/lookup/env_domain.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            type: boolean
+            default: False
+
+    notes:
+        - Requires C(cdpy).
+'''
+
+
+EXAMPLES = '''
+- name: Retrieve the Freeipa domian for a single CDP Public Cloud Environment
```
Comment (jimright) [inline plugins/lookup/env_domain.py:44]: typo here: `domian` -> `domain`

File: plugins/lookup/env_domain.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  ansible.builtin.debug:
+    msg: "{{ lookup('cloudera.cloud.env_domain', 'example-env') }}"
+
+- name: Retrieve the Freeipa domian and Private Freeipa host ips for a CDP Public Cloud Environment
```
Comment (jimright) [inline plugins/lookup/env_domain.py:48]: typo here: `domian` -> `domain`

**Review by jimright — CHANGES_REQUESTED** [review]
Just a few comments on the `env_domain` plugin. Otherwise all looks good.

File: plugins/lookup/env_domain.py (hunk 2 above)
Comment (rsuplina) [reply plugins/lookup/env_domain.py:48]: Good spot , replaced

File: plugins/lookup/env_domain.py (hunk 1 above)
Comment (rsuplina) [reply plugins/lookup/env_domain.py:44]: Replaced

File: plugins/lookup/env_domain.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    description:
+        - Allows you to retrieve information about Freeipa Domain for a given CDP Public Cloud Environment.
+        - If the Environment is not found or is ambigious, the lookup will return an error.
+    options:
+        _terms:
+            description:
+                - A CDP Public Cloud Environment name
+            type: string
+            required: True
+        detailed:
```
[the remark this answers is not in the document]
Comment (rsuplina) [reply plugins/lookup/env_domain.py:31]: If env_domain got "detailed=True" it will return Private IP's.

**Pull request closed without merging**